src/datatypes/src/schema/column_schema.rs

```diff
@@ -43,6 +44,34 @@ pub struct ColumnSchema {
     metadata: Metadata,
 }
 
+impl fmt::Debug for ColumnSchema {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        let mut formatted = format!(
+            "{} {} {}",
+            self.name,
+            self.data_type,
+            if self.is_nullable {
+                "nullable"
+            } else {
+                "not null"
+            },
+        );
+
+        // Add default constraint if present
+        if let Some(default_constraint) = &self.default_constraint {
+            formatted.push_str(&format!(" default={:?}", default_constraint));
```

Comment:
Use `write` to avoid constructing temporary strings.
```suggestion
            write!(f, " default={:?}", default_constraint)?;
```